server/src/test/java/org/elasticsearch/common/UUIDTests.java

```diff
@@ -152,6 +153,7 @@ protected long currentTimeMillis() {
         // the quality of this test
         Directory dir = newFSDirectory(createTempDir());
         IndexWriterConfig config = new IndexWriterConfig()
+                .setCodec(Codec.forName("Lucene84"))
```

Comment:
Should we use `Lucene.LATEST_CODEC` here so we don't need to update on every lucene bump in multiple places?